hs_discover/src/components/Resources.vue

```diff
@@ -0,0 +1,752 @@
+<!--                                data-toggle="tooltip" data-placement="right" :title="entry.shareable?'Shareable':'Not Shareable'"-->
+<!--                                style="cursor:pointer" data-original-title="Shareable">-->
+                        </td>
+                        <td>
+                            <a :href="entry.link" data-toggle="tooltip" target="_blank" style="cursor:pointer"
+                               :title="ellip(entry.abstract)" data-placement="top">{{entry.title}}</a>
+                        </td>
+                        <td>
+                            <a :href="entry.author_link" data-toggle="tooltip" target="_blank" style="cursor:pointer"
+                               :title="`Author: ${entry.author}
+
+Owner: ${entry.owner}
+
+Contributor: ${entry.contributor}`">{{entry.author}}</a>
+<!-- Ensure the literal line above is not spaced or those spaces will appear in the tooltip -->
+                        </td>
+                        <!-- python is passing .isoformat() in views.py -->
+                        <td style="cursor:pointer" data-toggle="tooltip" :title="new Date(entry.created).toLocaleTimeString('en-US')">{{new Date(entry.created).toLocaleDateString('en-US')}}</td>
+                        <td style="cursor:pointer" data-toggle="tooltip" :title="new Date(entry.created).toLocaleTimeString('en-US')">{{new Date(entry.modified).toLocaleDateString('en-US')}}</td>
+                    </tr>
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+    </div>
+</template>
+
+<script>
+import DatePick from 'vue-date-pick';
+import 'vue-date-pick/dist/vueDatePick.css';
+import axios from 'axios'; // css font-size overridden in hs_discover/index.html to enforce 1em
+
+export default {
+  data() {
+    return {
+      resloaded: false,
+      resources: [],
+      searchtext: '',
+      filterlimit: 10, // Minimum threshold for filter item to display with checkbox
+      geodata: [],
+      geopoints: [],
+      startdate: 'Start Date',
+      enddate: 'End Date',
+      filteredcount: 0,
+      perpage: 40, // resources per page
+      pagenum: 1, // initial page number to show
+      geoloaded: false, // searchjson endpoint called and retrieved geo data
+      resgeotypes: '',
+      googMarkers: [],
+      uidFilter: [],
+      countAuthors: {},
+      authorFilter: [],
+      countSubjects: {},
+      subjectFilter: [],
+      countOwners: {},
+      ownerFilter: [],
+      countContributors: {},
+      contributorFilter: [],
+      countTypes: {},
+      typeFilter: [],
+      countAvailabilities: {},
+      availabilityFilter: [],
+      sortDir: -1, // 1 asc -1 desc
+      sortingBy: 'modified',
+      resIconName: {
+        'Composite Resource': '/static/img/resource-icons/composite48x48.png',
+        Generic: '/static/img/resource-icons/generic48x48.png',
```

Comment:
This is inconsistent with the reset of the resIconName keys

Reply:
We just discussed this and it's the eslint standard